### PR TITLE
Improve scapi.h documentation

### PR DESCRIPTION
**Pull request by wwahammy**
Signed-off-by: Eric Schultz <eschultz@prplfoundation.org>

### CODE REVIEW

File: scapi.h [hunk 1]
```diff
@@ -160,14 +160,16 @@ struct scapi_plugin {
...
+	 * Validate all changes (optional)
 	 *
 	 * ptr: reference to plugin, data model
 	 */
 	int (*validate)(struct scapi_ptr *ptr);
 
 	/*
-	 * Commit all changes
+	 * Commit all changes (optional)
+   * If your plugin directly modifies data without a commit stage, this isn't needed
```
Comment (lperkov) [inline scapi.h]: Please fix whitespace :)

Comment (wwahammy) [reply scapi.h]: ha, oops, will do :)

**Comment by nbd168** [comment]
Formatting is still wrong in some places, on continuation of multi-line comments there should be a space character before `*`
Please use git rebase to merge your commits to have a clean history.
Thanks

**Comment by wwahammy** [comment]
Alright, I've edited and squashed commits. I don't think there are any formatting issues but if there are, please let me know.

**Comment by nbd168** [comment]
Applied, thanks.

**Pull request closed without merging**